projects/sandbox/src/app/home/home.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="home">
-  <div class="home-header">
-    <h1>Welcome to<br />Metis Sandbox!</h1>
-    <div class="circle-container-outer">
-      <ul class="circle-container">
+  <div class="wizard-head">
```

Comment:
I've seen the word `wizard` being used in multiple places, what is this referring to?
For me it is not clear/obvious what this is meant to represent

Reply:
> I've seen the word `wizard` being used in multiple places, what is this referring to? For me it is not clear/obvious what this is meant to represent

This is legacy nomenklature.  I've created [this ticket](https://europeana.atlassian.net/browse/MET-5008) to remove it